flight/targets/sparky2/fw/pios_board.c

```diff
@@ -683,13 +686,38 @@ void PIOS_Board_Init(void) {
 
 	PIOS_SENSORS_Init();
 
+	uint8_t dac_mode;
+	HwSparky2AdcDacGet(&dac_mode);
+	struct pios_internal_adc_cfg *adc_cfg = &pios_adc_withoutdac_cfg;
+
+	// Select what the ADC or DAC is used for
+	switch(dac_mode) {
+	case HWSPARKY2_ADCDAC_ADC:
+		adc_cfg = &pios_adc_withdac_cfg;
+		break;
+	case HWSPARKY2_ADCDAC_BEEP:
+#if defined(PIOS_INCLUDE_DAC_BEEPS)
+	{
+		uintptr_t dacbeep_id;
+		PIOS_DACBEEP_Init(&dacbeep_id);
+		dacbeep_handle = dacbeep_id;
+	}
+#endif /* PIOS_INCLUDE_DAC_BEEPS */
+		break;
+	case HWSPARKY2_ADCDAC_FSKTELEM:
+#if defined(PIOS_INCLUDE_FSK)
+	{
+		uintptr_t fskdac_com_id;
+		PIOS_FSKDAC_Init(&fskdac_com_id, &pios_fskdac_config);
+	}
+#endif /* PIOS_INCLUDE_FSK */
```

Comment:
this one I'll skip because FSK_DAC isn't actually included in this build - it's avoiding me bumping the UAVO twice.